spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
Ok.  The only reason I did start testing it was because Code Climate said it wasn't covered.